crates/astria-conductor/src/executor/mod.rs

```diff
@@ -569,7 +571,7 @@ impl Executor {
     }
 
     // Returns the next sequencer block height which can be executed on the rollup
-    pub(crate) fn get_executable_block_height(&self) -> Result<u32> {
+    pub(crate) fn calculate_executable_block_height(&self) -> Result<tendermint::block::Height> {
```

Comment:
These (this one and the one below) are only ever used as tendermint `Height`s, so it's silly to convert to/from u32 all the time.